src/lib/models/reflections/container.ts

```diff
@@ -45,47 +45,4 @@ export class ContainerReflection extends Reflection {
             }
         }
     }
-
-    /**
-     * Return a raw object representation of this reflection.
-     * @deprecated Use serializers instead
-     */
-    toObject(): any {
```

Comment:
It is possible, yes, but I'm not sure it is a good idea. To make it work we have 2 options.

1. Construct a new serializer whenever `.toObject` is called
1. Add an `application: Application` property to every reflection so that it can access the real serializer.

There are problems with both. 

1. The new serializer will be missing any extra serializers added by plugins
1. Requires changing every reflection, and reflections really shouldn't know about serializers anyways (at least, that's the theory with having separate serialization...)

Since `.toObject` has been deprecated for nearly 2 years, I feel pretty ok about just removing the method.